File: monty-hall-problem-demo/monty-hall-problem-demo/Models/AttemptResultReseponse.cs
namespace monty_hall_problem_demo.Models
{
    public class AttemptResultReseponse
    {
        public List<GameCardInfo> GameCardInfoList { get; set; }
        public int NumberOfGameAttempt { get; set; }
        public int CountOfWon { get; set; } = 0;
        public int CountOfLost { get; set; } = 0;
        public double PrecentageofCountOfWon { get; set; } = 0.00;
        public double PrecentageofCountOfLost { get; set; } = 0.00;
    }
}

File: monty-hall-problem-demo/monty-hall-problem-demo/ClientApp/src/app/game/models/AttemptResultReseponse.ts
import { GameCardInfo } from "./GameCardInfo";

export interface AttemptResultReseponse {
    gameCardInfoList: GameCardInfo[];
    numberOfGameAttempt: number;
    countOfWon: number;
    countOfLost: number;
    precentageofCountOfWon: number;
    precentageofCountOfLost: number;
}